xmake/modules/detect/tools/gdc/has_flags.lua

```diff
@@ -11,13 +11,113 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
--- 
+--
 -- Copyright (C) 2015-2020, TBOOX Open Source Group.
 --
--- @author      ruki
+-- @author      ruki, BarrOff
 -- @file        has_flags.lua
 --
 
 -- imports
-inherit("detect.tools.dmd.has_flags")
+import("lib.detect.cache")
```

Comment:
I don't see the difference with `detect.tools.dmd.has_flags·, why not reuse it directly?